packages/cli-lib/modules.js

```diff
@@ -4,6 +4,11 @@ const { getCwd, getExt, splitHubSpotPath, splitLocalPath } = require('./path');
 const { walk } = require('./lib/walk');
 const { MODULE_EXTENSION } = require('./lib/constants');
 
+// Matches files named module.html
+const MODULE_HTML_EXTENSION_REGEX = new RegExp(/(\.module\/module\.html)/);
```

Comment:
Nit, but I'm pretty sure this would also match if there was something after the `.html` extension. For example, `my-module.module/module.htmlanythingelse`